src/plugins/intel_gpu/docs/basic_data_structures.md

```diff
@@ -96,7 +96,7 @@ An example creation of a `arg_max_min` primitive:
 cldnn::arg_max_min top_k_prim = cldnn::arg_max_min("top_k", { "input" }, arg_max_min::max, top_k, arg_max_min::y, arg_max_min::sort_by_values, false, "", padding(), data_types::f32);
 ```
 
-In GPU plugin, the *primitives* are converted from ngraph [operations](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_gpu/src/plugin/ops).
+In GPU plugin, the *primitives* are converted from OpenVINO opset [operations](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_gpu/src/plugin/ops).
```

Comment:
nit: I think OpenVINO is enough (without opset) or from OpenVINO opsets without operations 😄 .

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/18920/commits/2367f0a1959c79a43294f77c8860071463a73a9c